experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -335,6 +335,9 @@ class Meta:
     def save(self, commit=True):
         experiment = super().save(commit=commit)
         experiment.subscribers.add(self.request.user)
+        generate_nimbus_changelog(
+            experiment, self.request.user, self.get_changelog_message()
+        )
```

Comment:
Why add this here?  This is already done by `NimbusChangeLogFormMixin`?

Reply:
oh if I am overriding the save method, I noticed that it is not generating the changelogs, so for that, I needed to explicitly called the generate changelogs method